src/Parallel/Algorithm.hpp

```diff
@@ -222,6 +222,32 @@ class AlgorithmImpl<ParallelComponent, tmpl::list<PhaseDepActionListsPack...>>
   template <typename Action>
   void simple_action() noexcept;
 
+  /// \brief Perform a blocking call the action `Action` with the current
```

Comment:
I don't know what you mean by `blocking` here. To me "blocking" means that there's some parallel data dependency with an explicit "wait until you've received this". I guess that's sort of the case, but I think it would be clearer to say something along the lines of "Call the Action synchronously, returning a result without any parallelization. That is, the action is called immediately and control flow returns to the caller immediately upon completion." This is pretty wordy, but I think the thing to really emphasize is that it's a call that happens on a local object, no threading, no MPI, etc. I think the other thing to make clear is that this doesn't block other actions/entry methods on a nodegroup